aspnet-core/xml/ns-Microsoft.AspNetCore.Cryptography.KeyDerivation.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Cryptography.KeyDerivation">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types for algorithms for performing key derivation.</summary>
+    <remarks>For more information, see <see href="https://docs.microsoft.com/en-us/aspnet/core/security/data-protection/introduction">Data Protection docs</see>.</remarks>
```

Comment:
Updating links to remove language from URL. Allows it to be auto-detected from browser
```suggestion
    <remarks>For more information, see <see href="https://docs.microsoft.com/aspnet/core/security/data-protection/introduction">Data Protection docs</see>.</remarks>
```